apps/frontline-widgets/src/app/messenger/components/conversation.tsx

```
@@ -0,0 +1,248 @@
+export function EmptyChat() {
+  const connection = useAtomValue(connectionAtom);
+  const { messengerData } = connection?.widgetsMessengerConnect || {};
+  const { responseRate } = messengerData || {};
+  const { list } = useGetMessengerSupporters();
+  return (
+    <div className="flex flex-col gap-4 font-medium text-sm">
+      <div className="my-auto flex items-center gap-2">
+        <AvatarGroup max={3} size="md">
+          {list &&
+            list?.map((supporter: ISupporter) => (
+              <Avatar className="border-2 border-sidebar size-10" size="xl">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)